liquibase-standard/src/main/java/liquibase/util/ShowSummaryUtil.java

```diff
@@ -262,12 +262,14 @@ private static UpdateSummary showSummary(DatabaseChangeLog changeLog,
             builder.append(System.lineSeparator());
         }
 
-        try {
-            Writer writer = createOutputWriter(outputStream);
-            writer.append(builder.toString());
-            writer.flush();
-        } catch (IOException ioe) {
-            throw new LiquibaseException(ioe);
+        if (verbose) {
```

Comment:
Earlier talking with a team member he pointed out this is the piece that is making fail a couple of update tests (`update`, `updateCount` and `updateToTag`) where a `showSummary` is set with a different value than `verbose` and this is causing to not make write the command execution result. Could you please take a look at this?

Thanks,
Daniel.

Reply:
Thank you, I changed the level to verbose to all the failing tests and now they are working!

Reply:
It's because the summary logs are being outputted to the "standard" log (I don't how you call it sorry) with `Scope.getCurrentScope().getLog(ShowSummaryUtil.class).info()` and printed directly to the standard output. The if remove the printing to the standard output, but not to the standard log. The test is set up to redirect the standard output to the file, so unless it is set to verbose it seems to not write anything.

Should I try to redirect the standard log to the file to check if it's logging correctly? Are there already places where it's done so I can copy them?

Reply:
I'll answer myself: it wasn't at all difficult thanks to the `expectedLogs` property. 

I added to the `update.test` all the summary and verbose level tests to ensure that they behaves correctly. Does this resolve your doubts @MalloD12? I can update the `updateCount.test` and `updateToTag.test` too if you think it's fine.

Reply:
Hi @EvaristeGalois11, thank you for updating these tests. I think it would be nice to update the others as well to keep them consistent, but let me check with somebody else from the team to double-check others are happy with it as well. I'll get back to you soon.